enhancements/network/egress-qos.md

```diff
@@ -27,8 +29,9 @@ tracking-link:
 Not all traffic has the same priority, and when there is contention for bandwidth, there should be a mechanism for objects outside the cluster to prioritize the traffic.
 To enable this, we will use Differentiated Services Code Point (DSCP) which allows us to classify packets by setting a 6-bit field in the IP header, effectively marking the priority of a given packet relative to other packets as "Critical", "High Priority", "Best Effort" and so on.
 
-By introducing a new CRD `EgressQoS`, users could specify a DSCP value for packets originating from pods on a given namespace heading to a specified CIDR.
+By introducing a new CRD `QoS`, users could specify a DSCP value for packets originating from pods on a given namespace heading to a specified CIDR, Protocol and Port.
 The CRD will be Namespaced, with one resource allowed per namespace.
+The CRD is also designed to be so generic enough so that it can support DSCP QoS for ingress traffic and traffic policing/shaping in future.
```

Comment:
Please add edits to the motivation section for this.
Also @trozet mentioned we don't plan to add new features like policing/shaping which is why this is present in the nonGoals section.

So if the goal is only "extensibility" before GA that's fine in terms of the "API design" being a non-goal, but I think the motivation/use cases must be solid for this renaming.

Reply:
yes, modified motivation and non goals sections now, is this fine now ?